demo-app/Brewfile

```diff
@@ -0,0 +1,2 @@
+brew "overmind"
```

Comment:
fwiw I don't think we're using `overmind`, but I would be down to do so!

Reply:
👍 to overmind, especially if we eventually want to setup background jobs.